tests/Homely.AspNetCore.Mvc.Helpers.Tests/TestFixture.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Testing;
+using TestWebApplication;
+
+namespace Homely.AspNetCore.Mvc.Helpers.Tests
+{
+    public class TestFixture : WebApplicationFactory<Startup>
+    {
+        protected override void ConfigureWebHost(IWebHostBuilder builder)
+        {
+            base.ConfigureWebHost(builder);
+
+            builder.UseEnvironment("Production");

```

Comment:
why Production and not 'Test'? could cause some side effects? also, isn't there an enum?

Reply:
So the json output doesn't include the exception information. `IsDevelopment()` shows stack trace.

As to an ENUM ? I didn't think there is/was ... ???

/me checks.

The best I could find is [this MSDN docs on Environments](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/environments?view=aspnetcore-3.1). There's code to _check_ the environment, but I couldn't find any code to SET it. 

It's usually set via:
- config file
- env vars.

Reply:
maybe im confused, so i'll ask a diff way.. _why_ are you explicitly setting the environment to Production in a test?

Reply:
so the result JSON does not include an exception stack trace.

> So the json output doesn't include the exception information. IsDevelopment() shows stack trace.

Reply:
ok cool..so for your tests, you don't _want_ the exception stack trace.  so you set environment to "Production".

next question: why don't you want the exception stack trace? is you change to "Test" here, will some tests fail?

Reply:
Resolved after slack-chat. All good.